homeassistant/components/motionblinds_ble/strings.json

```diff
@@ -20,6 +20,18 @@
       }
     }
   },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Connection options",
+        "description": "**WARNING**: Changing any of the below options may significantly reduce battery life of your motor!",
+        "data": {
+          "permanent_connection": "Permanent connection",
+          "disconnect_time": "Disconnect time (seconds)"
```

Comment:
The permanent connection option makes sense given its BLE.

For the disconnect time, why would the user need to change this? How does the user know what a good value is here?

Reply:
> For the disconnect time, why would the user need to change this? How does the user know what a good value is here?

It defaults to 15 seconds which I would say is a reasonable time for normal-sized blinds, and it gets refreshed every time an action is performed (like opening, closing of the blind etc). However, I can imagine some users would like it to be a little higher, maybe because their blinds are bigger thus requiring more time to open/close fully. Also, it is an *option*, and I would say it is never bad to have the option to change it to your liking

See this related issue:
https://github.com/home-assistant/core/issues/120105

Reply:
Please add some some descriptive text explaining why you would want to increase the value.

Something like:

The default disconnect time is 15 seconds. If your shades take longer to perform an operation, increase the disconnect time by counting how many seconds it takes for a full open-to-close operation to be performed and adding 5 seconds.

Reply:
Where and how can I add that? Is it possible to add a description to each of the options?

Reply:
Usually thats accomplished by extending the `description` field or for less important options, its added to the docs (but users don't usually see thoose)

https://github.com/home-assistant/core/pull/120110/files#diff-67d46204179e211a11e8bffe64c02f3799c7f8bbcf8a1248013bf6b232f3691cR27

Reply:
What about adding it to the `description` of `vol.Optional`, or won't it be translated in that case?

Reply:
Updated the description field to:

The default disconnect time is 15 seconds, adjustable using the slider below. You may want to adjust this if you have larger blinds or other specific needs. You can also enable a permanent connection to the motor, which disables the disconnect time and automatically reconnects when the motor is disconnected for any reason.
**WARNING**: Changing any of the below options may significantly reduce battery life of your motor!